integration_tests/mockApis/lostBed.ts

```diff
@@ -21,6 +21,55 @@ export default {
       },
     }),
 
+  stubLostBedUpdate: (args: { lostBed: LostBed; premisesId: string }): SuperAgentRequest =>
+    stubFor({
+      request: {
+        method: 'PUT',
+        url: `/premises/${args.premisesId}/lost-beds/${args.lostBed.id}`,
+      },
+      response: {
+        status: 201,
+        headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+        jsonBody: args.lostBed,
+      },
+    }),
+
+  stubLostBedUpdateErrors: (args: { lostBed: LostBed; premisesId: string; params: Array<string> }): SuperAgentRequest =>
+    stubFor({
+      request: {
+        method: 'PUT',
```

Comment:
had to use this instead of the ready made `errorStub` because it's a `PUT` instead of a `POST`